app-frontend/packs/components/dashboard/homePage/HomePageView.vue

```diff
@@ -1,19 +1,48 @@
 <template>
 
-    <LayoutPage>
-        <HomePage></HomePage>
-    </LayoutPage>
+    <div v-if="!$store.state.isLoader">
+
+        <HomePageEmptyBanner
```

Comment:
```suggestion
        <HomePageEmptyState
```

лучше переименовать, а то выгдяжит так, как буд-то хоум пейджа с пустым баннером

Reply:
исправил